go/chat/search/storage.go

```diff
@@ -46,17 +48,32 @@ func newStore(g *globals.Context) *store {
 	dbFn := func(g *libkb.GlobalContext) *libkb.JSONLocalDb {
 		return g.LocalChatDb
 	}
+	nlru, err := lru.New(1000)
```

Comment:
Are you sure this isn't too big?

Reply:
could dial it back a bit if we see an issue on mobile but was hoping we fit most inboxes since otherwise the cache is useless and we'll just thrash on searches

Reply:
dropped down to 250 and moved to the indexer instead of storage so we just use it on searches